src/ui/components/Timeline/Timeline.css

```diff
@@ -33,6 +33,10 @@
   font-size: 29px;
 }
 
+.timeline.show-protocol-timeline {
+  padding-top:40px;
```

Comment:
This feels a bit janky from a layout perspective, but I also don't want to be a perfectionist, especially for what is basically an internal diagnostic tool 🔨

Reply:
hehe totally. This is a surprisingly hard problem

Reply:
:cough:Tailwind:cough: 😇